src/cgame/rocket/rocket.cpp

```diff
@@ -140,6 +140,14 @@ class DaemonSystemInterface : public Rml::Core::SystemInterface
 
 	void SetMouseCursor(const Rml::Core::String& cursor_name) override
 	{
+		if ( cursor_name.empty() )
```

Comment:
Maybe it's somehow possible to prevent the cursor's name to get empty, instead?
Also, this commit looks like it could be squashed with e61b3a38 ?

Reply:
This clearly would feel like dirty hack. Maybe in later rmlui version it's possible to get a default cursor... we'll see I guess.